webpack/set-webpack-public-path-plugin/src/SetPublicPathPlugin.ts

```diff
@@ -209,7 +200,7 @@ export class SetPublicPathPlugin implements Webpack.Plugin {
     );
 
     // Webpack 5 has its own automatic public path code, so only apply for Webpack 4
-    if (webpackMajorVersion === 4) {
+    if (VersionDetection.isWebpack4) {
```

Comment:
```suggestion
    if (VersionDetection.isWebpack4(compiler)) {
```

Maybe just stick this in a `isWebpack4` variable.